src/components/LabelWithInfo.tsx

```diff
@@ -14,11 +14,13 @@ export function LabelWithInfo({
   iconSize = 16,
   testID,
   style,
+  numberOfLines,
 }: {
   label: string
   onPress?: () => void
   labelStyle?: StyleProp<TextStyle>
   iconSize?: number
+  numberOfLines?: number | undefined
```

Comment:
I wonder if it's better to have `1` or `undefined` as the default

Reply:
When I tried to set the default to 1 I ran into an issue where setting it to undefined wasn't overriding the default value. I can try setting the default to undefined and see how that works.